RecoParticleFlow/PFTracking/python/mergedElectronSeeds_cfi.py

```diff
@@ -1,9 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 
 
-
 electronMergedSeeds =cms.EDProducer("ElectronSeedMerger",
      EcalBasedSeeds = cms.InputTag("ecalDrivenElectronSeeds"),
      TkBasedSeeds  = cms.InputTag("trackerDrivenElectronSeeds:SeedsForGsf")
     )
 
+electronMergedSeedsFromMultiCl = electronMergedSeeds.clone()
+
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+phase2_hgcal.toModify(
+  electronMergedSeedsFromMultiCl,
```

Comment:
era is not needed here either, just pass the parameters to the .clone (as already done in a few other places, Thanks.)